From dc77d65c8554c2dbc00d3e4f73f8ee3bc8380c77 Mon Sep 17 00:00:00 2001
From: Xin Li <talons.lee@gmail.com>
Date: Tue, 9 Oct 2018 17:33:18 +0800
Subject: [PATCH v2 4.9 12/20] xen/xsm: remove unnecessary #define

this #define is unnecessary since XSM_INLINE is redefined in
xsm/dummy.h, it's a risk of build breakage, so remove it.

Signed-off-by: Xin Li <xin.li@citrix.com>
Reviewed-by: Jan Beulich <jbeulich@suse.com>
Acked-by: Daniel De Graaf <dgdegra@tycho.nsa.gov>
---
 xen/xsm/dummy.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/xen/xsm/dummy.c b/xen/xsm/dummy.c
index 3cb5492dd3..667078ea07 100644
--- a/xen/xsm/dummy.c
+++ b/xen/xsm/dummy.c
@@ -11,7 +11,6 @@
  */
 
 #define XSM_NO_WRAPPERS
-#define XSM_INLINE /* */
 #include <xsm/dummy.h>
 
 struct xsm_operations dummy_xsm_ops;
-- 
2.17.1

